components/server/ee/src/workspace/gitpod-server-impl.ts

```diff
@@ -250,6 +250,29 @@ export class GitpodServerEEImpl extends GitpodServerImpl {
     ): Promise<void> {
         await super.mayStartWorkspace(ctx, user, runningInstances);
 
+        {
+            // dummy implementation to test frontend bits
+            const attributionId = user.usageAttributionId;
+            const costCenter = !!attributionId && (await this.costCenterDB.findById(attributionId));
+            if (costCenter) {
```

Comment:
Even if this (atm) guards against UBB being enabled: I think we should ask the user's feature flag here, for now, so we can merge and deploy this to prod. :+1:

Reply:
'added FF-check.